deps/v8/src/heap/base/asm/ppc/push_registers_asm.cc

```diff
@@ -33,7 +33,7 @@ asm(
     // At anytime, SP (r1) needs to be multiple of 16 (i.e. 16-aligned).
     "  mflr 0                                          \n"
     "  std 0, 16(1)                                    \n"
-#if defined(_AIX)
+#if ABI_USES_FUNCTION_DESCRIPTORS
```

Comment:
`ABI_USES_FUNCTION_DESCRIPTORS` is not defined in this file, no V8 headers are included here, it's raw assembly, hence `defined(_AIX)` is used to rely on gcc.

Reply:
could use `!defined(_CALL_ELF) || (_CALL_ELF == 1)` for a generic check